### PR TITLE
[KOGITO-161] - Upgrade Kogito Operator image to ubi8

### PR DESCRIPTION
**Pull request by xiezhang7**
Jira: https://issues.jboss.org/browse/KOGITO-161

Description: use base image registry.access.redhat.com/ubi8-minimal for Kogito operator.

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by ricardozanini** [comment]
@spolti could you take a look into this upgrade? Since you worked on the runtimes images, you might have some tips.

Many thanks!

**Comment by xiezhang7** [comment]
Hi @ricardozanini @bmozaffa @tchughesiv , Could you please review this pull request, too? Mainly, I did what was done to the kie-cloud-operator. Thanks.

File: build/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,11 @@
-FROM registry.access.redhat.com/ubi7-dev-preview/ubi-minimal:7.6
-
-ENV OPERATOR=/usr/local/bin/kogito-cloud-operator \
-    USER_UID=1001 \
-    USER_NAME=kogito-cloud-operator
+FROM registry.access.redhat.com/ubi8-minimal
```
Comment (spolti) [inline build/Dockerfile]: please change to registry.redhat.io/ubi8-minimal:latest

Comment (xiezhang7) [reply build/Dockerfile]: Thanks, @spolti . I have changed the base image.

**Pull request merged**